src/DetailsView/components/details-view-right-panel.tsx

```diff
@@ -1,7 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import * as React from 'react';
```

Comment:
We dont need `react` for a `tsx` file? Just curious. 😄

Reply:
You only need to import `react` if you're using `jsx` syntax (like `return <Component .../>` which is not the case on this file. I didn't want to change the file extension as this will affect more files with a change that is not strictly "remove unused vars and imports"

Reply:
@Shobhit1 on second though... I'll try the change an see how it looks

Reply:
Updated!

Reply:
Thanks for taking this up. I think its just easier to comprehend a file without `JSX` as a `ts` file than a `tsx`.